File: Data Structures & Algorithms/concatenation-of-array/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    getConcatenation(nums) {
        const ans = []
        let repeataion = 2;
        while(repeataion !== 0){
        for(let i = 0; i < nums.length; i++){
            ans.push(nums[i]);
        }
        repeataion--;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/remove-element/submission-2.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int start = 0;
        int end = nums.size() - 1;
        while(start <= end){
            if(nums[start] == val){
                swap(nums[start], nums[end--]);
            }
            else{
                start++;
            }
        }
        return end+1;
    }
};